### ISSUE TITLE
Fail Gracefully when Waveform Information is not Available

### ISSUE THREAD

**Issue by Dananji** [avalonmediasystem/react-structural-metadata-editor#200]
### Description

SME reads `waveform.json` information from the `seeAlso` property of the Canvas, and when it is not provided SME fails with multiple `No available waveform data` message and keep blocking the UI as follows;

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOTdTQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7573c7b34d8ecce5eebafdd1bb78e3251e083ab6/image.png)

### Done Looks Like
- [x] Shows the relevant message in the UI as non-dismissible alert (without a close button)
- [x] Unblocks the UI when media is loaded into the component

**Comment by Dananji** [comment]
This was an issue with Peaks.js trying to start its initialization before SME properly sets up the `mediaElement` information on page with the stream media using `hls.js`

**Comment by elynema** [comment]
@Dananji Any suggestions on how to identify a good item on avalon-dev to use for QA for this? In general, I'm able to load SME without seeing this error, but not sure if I've checked a file that doesn't have a waveform.

The image above shows a media item that isn't on avalon-dev.

**Comment by elynema** [comment]
Per Dananji, SME doesn’t handle the empty Canvas use-case in its Manifest parsing, which is what we are seeing in this item. We need to do that work in SME.

See https://github.com/avalonmediasystem/react-structural-metadata-editor/issues/203 for new issue for this problem.
